config/esp32/components/chip/idf_component.yml

```diff
@@ -41,3 +41,5 @@ dependencies:
         version: ">=0.1.12"
         rules:
             - if: "target in [esp32p4]"
+
+espressif/esp_diag_data_store: "1.0.1"
```

Comment:
Hi @shripad621git Should we pin the version of esp_diag_data_store here or esp_insights?

Reply:
The above change is not required. We have pinned the insights versions [here](https://github.com/espressif/esp-insights/commit/a417202e7f2dce53853f46a87393a614e17b32e3) and we are using the latest esp-insights i.e 1.2.2.
The flow with insights enabled is tested with 1.2.2.

Reply:
@feasel0 PTAL